CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+
+cmake_minimum_required(VERSION 3.12.3)
+
+project(SQLite
+  LANGUAGES C)
```

Comment:
nit: should we add `VERSION 3.46.0`?

Reply:
We could, but that is just metadata and wouldn't be used. Future updates to the package would just then go out of sync unless we had use of the version number somewhere.